pinot-common/src/test/java/org/apache/pinot/common/data/DateTimeFormatSpecTest.java

```diff
@@ -82,6 +86,8 @@ public void testFromMillisToFormat(String format, long timeMs, String expectedFo
     entries.add(new Object[]{"1:MILLISECONDS:EPOCH", 1498892400000L, "1498892400000"});
     entries.add(new Object[]{"1:HOURS:EPOCH", 0L, "0"});
     entries.add(new Object[]{"5:MINUTES:EPOCH", 1498892400000L, "4996308"});
+    entries.add(new Object[]{"1:MILLISECONDS:TIMESTAMP", Timestamp
```

Comment:
Would be good to see a test case for `1:NANOSECONDS:TIMESTAMP` specially since Pinot supports `NANOSECONDS` and `java.sql.Timestamp` seems to require a separate call to `java.sql.Timestamp.setNanos(...)` to set nanoseconds in timestamp.

Reply:
SQL Timestamp always have values in `yyyy-MM-dd HH:mm:ss.SSS` format, and it doesn't allow `nanoseconds` granularity. The first 2 values are actually ignored (similar to date time format which only reads the 4th argument as the format). We should consider changing the order for these arguments for clarity, but that is out of the scope of this PR.